src/keysim.js

```diff
@@ -219,6 +221,14 @@ export class Keyboard {
         if (events & KeyEvents.INPUT) {
           const textinputEvent = this.createEventFromKeystroke('textInput', keystroke, target);
           target.dispatchEvent(textinputEvent);
+          const inputEvent = this.createEventFromKeystroke('input', keystroke, target);
+          target.dispatchEvent(inputEvent);
+        }
+      }
+      if (keystroke.keyCode === US_ENGLISH_ACTION_KEYCODE_MAP.BACKSPACE) {
```

Comment:
I think using `US_ENGLISH_ACTION_KEYCODE_MAP.BACKSPACE` here is bad, but I couldn't find the way to access the expected keycode in this scope 😓